src/__tests__/autocapture.test.ts

```diff
@@ -202,19 +266,20 @@ describe('Autocapture system', () => {
             expect(props['on-the-sensitive-input']).toBeUndefined()
         })
 
-        it('should collect augment from the hidden element value', () => {
+        it('should not collect augment from the hidden element value', () => {
             const props = autocapture._getAugmentPropertiesFromElement(hidden)
-            expect(props['on-the-hidden']).toBe('is on the hidden')
+            expect(props).toStrictEqual({})
         })
 
         it('should collect augment from the password element value', () => {
             const props = autocapture._getAugmentPropertiesFromElement(password)
-            expect(props['on-the-password']).toBe('is on the password')
+            expect(props).toStrictEqual({})
         })
     })
 
```

Comment:
These tests were wrong because we were setting input properties on a div, we shouldn't be capturing anything on hidden and password inputs. It behaves correctly, just the tests were confused